routing.rst

```diff
@@ -560,7 +626,7 @@ placeholder:
             /**
              * @Route("/blog/{page<\d+>?1}", name="blog_list")
```

Comment:
Unless I'm missing something, we never documented the `<\d+>` part of this syntax (and there is an inconsistency between the different code blocks in this configuration-block, the XML format has a space between route name and regex.

I'm not sure, but I think I would prefer to document only one of the two versions in the "main text" either the complete inline or the `requirements`/`defaults` block. We can then quickly tell about the other format in a small tip block (e.g. "If your requirements/defaults become too complex, you can also use the more verbose syntax and separate requirements/defaults from the match string:")

Reply:
Yes, it's explained in the doc. IT's a bit long, so it's hard to explain everything ... but we show the expanded format always, then we show the inlined format one time and we warn users: if regexps are complex, the inline format is not very readable.